Gems/Atom/RHI/Registry/Platform/Windows/PlatformLimits.setreg

```diff
@@ -21,11 +21,13 @@
                         "$type": "AZ::DX12::PlatformLimitsDescriptor",
                         "DescriptorHeapLimits":
                         {
-                            "DESCRIPTOR_HEAP_TYPE_CBV_SRV_UAV": [1000000, 1000000],
+                            "DESCRIPTOR_HEAP_TYPE_CBV_SRV_UAV": [100000, 1000000],
```

Comment:
The default value in
PlatformLimitsDescriptor.cpp
void PlatformLimitsDescriptor::LoadPlatformLimitsDescriptor(const char* rhiName)
Should also be changed accordingly.